core/src/main/kotlin/com/exactpro/th2/read/db/core/MessageLoader.kt

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2023 Exactpro (Exactpro Systems Limited)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.exactpro.th2.read.db.core
+
+interface MessageLoader {
+    fun load(dataSourceId: DataSourceId, properties: Map<String, String>): TableRow?
+
+    companion object {
+        val NULL_RESULT = object : MessageLoader {
```

Comment:
Please, add `@JvmField` annotation. Event if we don't use this from Java the Kotlin compiler will still use getter method to access this field instead of using the field directly